Description: use system-installed node-gyp for building test modules
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2015-09-09
Forwarded: not-needed
--- a/Makefile
+++ b/Makefile
@@ -390,13 +390,8 @@
 # for generating headers. The list is manually synchronized with install.py.
 ADDONS_HEADERS_PREREQS := tools/install.py \
 	config.gypi common.gypi \
-	$(wildcard deps/openssl/config/*.h) \
-	$(wildcard deps/openssl/openssl/include/openssl/*.h) \
-	$(wildcard deps/uv/include/*.h) \
-	$(wildcard deps/uv/include/*/*.h) \
 	$(wildcard deps/v8/include/*.h) \
 	$(wildcard deps/v8/include/*/*.h) \
-	deps/zlib/zconf.h deps/zlib/zlib.h \
 	src/node.h src/node_api.h src/js_native_api.h src/js_native_api_types.h \
 	src/node_api_types.h src/node_buffer.h src/node_object_wrap.h \
 	src/node_version.h
@@ -417,7 +412,7 @@
 	$(filter-out test/addons/??_*/*.h, $(wildcard test/addons/*/*.h))
 
 ADDONS_PREREQS := test/addons/.headersbuildstamp \
-	deps/npm/node_modules/node-gyp/package.json tools/build_addons.py
+	tools/build_addons.py
 
 define run_build_addons
 env $(PYTHON) "$$PWD/tools/build_addons.py" --loglevel=$(LOGLEVEL) --headers-dir "$(ADDONS_HEADERS_DIR)" $1
